doc/source/development/contributing_codebase.rst

```diff
@@ -957,10 +957,6 @@ regarding :ref:`documentation <contributing_documentation>`.
 Further, to let users know when this feature was added, the ``versionadded``
 directive is used. The sphinx syntax for that is:
 
-.. code-block:: rst
-
-  .. versionadded:: 1.1.0
```

Comment:
I think we want to keep this as an example of how to add this sphinx syntax. Might be good to just update the version (and the sentence below)

Reply:
agreed, like `2.1.0` (and update the "This will put the text *New in version 1.1.0*" text below)

Reply:
Yeah, should have read in more detail